crawler/features_crawler.py

```diff
@@ -1093,23 +1165,77 @@ def crawl_interface(self):
 
             yield (feature_key, feature_attributes)
 
-    def _crawl_interface_counters(self):
+    def _crawl_interface_counters(self, interface):
+        try:
+            bytes_sent = interface.bytes_sent
+        except Exception as e:
```

Comment:
The old code had this because different versions of psvmi have differents names. But it's not really needed anymore as we just support one psvmi version. It makes our lives easier. I might be missing something. If you want to keep it, you should change it to `AttributeError` (i realize there are some other places that we catch Exception, but it shoudl be really an AttributeError).